pkg/helpers/aws.go

```diff
@@ -384,25 +386,31 @@ func CloudWatchLogsStream(ctx context.Context, cw cloudwatchlogsiface.CloudWatch
 
 	sleep := time.Duration(100 * time.Millisecond)
 
+	fmt.Printf("StartTime=%d\n", req.StartTime)
```

Comment:
Should keep the same prefixes all the way through, otherwise these lines can be tough to tell what they're associated with